batchupload/uploader.py

```diff
@@ -345,8 +356,14 @@ def main(*args):
         '(optional)\n'
         '\t-confirm Whether to output a confirmation after each upload '
         'attempt (optional)\n'
+        '\t-test Whether to do mock upload, simply outputting to commandline. '
+        '(optional)\n'
         '\t-nochunk Whether to turn of chunked uploading, this is slow '
-        'and does not support files > 100Mb (optional)\n'
+        'and does not support files > 100Mb (optional, type:FILES only)\n'
+        '\t-only:PATH to file containing list of urls to upload, skipping all '
+        'others. One entry per line. (optional, type:URL only)\n'
+        '\t-skip:PATH to file containing list of urls to skip, uploading all '
+        'others. One entry per line. (optional, type:URL only)\n'
```

Comment:
Codewise they aren't mutually exclusive (although the need to use both would require some weird circumstances).

If you provide the same list to both arguments you will of course have 0 files to upload. But if you e.g. only want to upload images of vases (only list) but skip all blue images (skip list) then combining the two would give you only the subset of non-blue vases as expected.